plugin/normalize.go

```diff
@@ -76,6 +77,7 @@ func (h Host) Normalize() string {
 // An error is returned on error
 // Deprecated: use OriginsFromArgsOrServerBlock or NormalizeExact
 func (h Host) MustNormalize() (string, error) {
+	log.Warning("Host.Normalize and Host.MustNormalize are Deprecated. Use OriginsFromArgsOrServerBlock or NormalizeExact.")
```

Comment:
This will confuse the ordinary user. I might say something like:

> An external plugin is using the deprecated x or y function. These will be removed in future versions of CoreDNS. The plugin should be updated to use a or b instead.

If we can identify the plugin from this location that would be even better.